datasource/get_watertemp.go

```diff
@@ -71,10 +71,9 @@ func GetLocationWithLowestTemperature(locations []WaterTemperatureSchema) WaterT
 }
 
 func WaterTemperatureSchemaToString(wts WaterTemperatureSchema) string {
-	return fmt.Sprintf("%s %f %s %s %d %f %f %s ", wts.Type, wts.Temp_water, wts.FormattedTime, wts.Alias, wts.Ts, wts.Latitude, wts.Longitude, wts.Gmap)
+	return fmt.Sprintf("%s %f degrees at %s %s %d %f %f %s ", wts.Type, wts.Temp_water, wts.FormattedTime, wts.Alias, wts.Ts, wts.Latitude, wts.Longitude, wts.Gmap)
 }
 
 func hasLowerTemperature(location1 WaterTemperatureSchema, location2 WaterTemperatureSchema) bool {
 	return location1.Temp_water < location2.Temp_water
 }
-
```

Comment:
A matter of taste, but I prefer an empty line at end of files. This is probably a setting in your IDE, that does it automatically